flink/v2.1/flink/src/main/java/org/apache/iceberg/flink/data/FlinkSchemaVisitor.java

```diff
@@ -29,9 +30,10 @@
 import org.apache.iceberg.types.Type;
 import org.apache.iceberg.types.Types;
 
-abstract class FlinkSchemaVisitor<T> {
+@Internal
+public abstract class FlinkSchemaVisitor<T> {
```

Comment:
why is this class public now? I didn't seem to see any new usage in this PR.

Reply:
Reverted